recipes/libjxl/all/conanfile.py

```diff
@@ -0,0 +1,126 @@
+from conans import ConanFile, CMake, tools
+import os
+import shutil
+import glob
+
+required_conan_version = ">=1.33.0"
+
+
+class LibjxlConan(ConanFile):
+    name = "libjxl"
+    description = "JPEG XL image format reference implementation"
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/libjxl/libjxl"
+    topics = ("image", "jpeg-xl", "jxl", "jpeg")
+
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+
+    exports_sources = "CMakeLists.txt", "patches/**"
+    generators = "cmake"
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def requirements(self):
+        self.requires("brotli/1.0.9")
+        self.requires("highway/0.12.2")
+        self.requires("lcms/2.11")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+                  destination=self._source_subfolder, strip_root=True)
+
+    def _patch_sources(self):
+        for patch in self.conan_data["patches"][self.version]:
+            tools.patch(**patch)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["BUILD_TESTING"] = False
+        self._cmake.definitions["JPEGXL_STATIC"] = not self.options.shared
+        self._cmake.definitions["JPEGXL_ENABLE_BENCHMARK"] = False
+        self._cmake.definitions["JPEGXL_ENABLE_EXAMPLES"] = False
```

Comment:
```suggestion
        self._cmake.definitions["JPEGXL_ENABLE_EXAMPLES"] = False
        self._cmake.definitions["JPEGXL_ENABLE_MANPAGES"] = False
```
By default is enabled